polyfactory/factories/attrs_factory.py

```diff
@@ -79,4 +79,4 @@ def resolve_types(cls, model: type[T], **kwargs: Any) -> None:
         :param kwargs: Any parameters that need to be passed to `attrs.resolve_types`.
         """
 
-        attrs.resolve_types(model, **kwargs)  # type: ignore[type-var]
+        attrs.resolve_types(model, **kwargs)
```

Comment:
@adhtruong is this not causing issues for you locally? `pre-commit` is complaining for me locally (on the defaults PR) :/

Reply:
Hmm strange. It may be an environment thing because I'm getting an error on main on this line (unused-ignore).

I'm using 3.11 but I'll double check against CI set up